ros/finalarm_control/launch/finalarm_gazebo_position_torque_control.launch

```diff
@@ -18,7 +18,7 @@
   <!-- convert joint states to TF transforms for rviz, etc -->
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
     respawn="false" output="screen">
-    <remap from="/joint_states" to="/finalarm/joint_states" />
+    <remap from="/joint_states" to="/finalarm_manager/joint_states" />
```

Comment:
Oops, this needs to get fixed in the same way that the position launch file was fixed; this was me figuring out how namespaces worked.